app/src/main/java/org/oppia/app/customview/inputInteractionView/NumberInputInteractionView.kt

```diff
@@ -0,0 +1,38 @@
+package org.oppia.app.customview.inputInteractionView
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.text.InputFilter
+import android.text.method.DigitsKeyListener
+import android.widget.EditText
+import org.oppia.app.R
+
+/** The customclass for [NumberInputInteractionView]. */
+@SuppressLint("ViewConstructor")
+class NumberInputInteractionView(context: Context, private var placeholder: String, private var rows: Int) :
+  EditText(context) {
+
+  private var mContext: Context = context
```

Comment:
For parameter names, prefer not prefixing with m per 
https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names.

Reply:
Resolved